src/Backend/Modules/MiniBlog/Engine/Model.php

```diff
@@ -0,0 +1,232 @@
+        $db = BackendModel::getContainer()->get('database');
+
+        // delete records
+        $db->delete('mini_blog', 'id = ?', $id);
+
+        // get used meta ids
+        $metaId = (int)$db->getVar(
+            'SELECT meta_id
+             FROM mini_blog AS i
+             WHERE id = ?',
+            array($id)
+        );
+
+        // delete meta
+        $db->delete('meta', 'id = ?', array($metaId));
+
+        // invalidate the cache for mini blog
+        BackendModel::invalidateFrontendCache('mini_blog', BL::getWorkingLanguage());
+    }
+
+    /**
+     * Checks if an item exists
+     *
+     * @param int $id
+     * @return bool
+     */
+    public static function exists($id)
+    {
+        return (bool)BackendModel::getContainer()->get('database')->getVar(
+            'SELECT i.id
+             FROM mini_blog AS i
+             WHERE i.id = ? AND i.language = ?',
+            array((int)$id, BL::getWorkingLanguage())
+        );
+    }
+
+    /**
+     * Get all data for a given id
+     *
+     * @param int $id
+     * @return array
+     */
+    public static function get($id)
+    {
+        return (array)BackendModel::getContainer()->get('database')->getRecord(
+            'SELECT i.*, UNIX_TIMESTAMP(i.created) AS created, UNIX_TIMESTAMP(i.edited) AS edited, m.url
+             FROM mini_blog AS i
+             INNER JOIN meta AS m ON m.id = i.meta_id
+             WHERE i.id = ? AND i.language = ?',
+            array((int)$id, BL::getWorkingLanguage())
+        );
+    }
+
+    /**
+     * Get all items by a given tag id
+     *
+     * @param int $tagId
+     * @return array
+     */
+    public static function getByTag($tagId)
+    {
+        $items = (array)BackendModel::getContainer()->get('database')->getRecords(
+            'SELECT i.id AS url, i.title AS name, mt.module
+             FROM modules_tags AS mt
+             INNER JOIN tags AS t ON mt.tag_id = t.id
+             INNER JOIN mini_blog AS i ON mt.other_id = i.id
+             WHERE mt.module = ? AND mt.tag_id = ? AND i.language = ?',
+            array('mini_blog', (int)$tagId, BL::getWorkingLanguage())
+        );
+
+        foreach ($items as $key => $row) {
+            $items[$key]['url'] = BackendModel::createURLForAction(
+                'edit', 'mini_blog', null,
+                array('id' => $row['url'])
+            );
+        }
+
+        return $items;
+    }
+
+    /**
+     * Get the maximum id
+     *
+     * @return int
+     */
+    public static function getMaximumId()
+    {
+        return (int)BackendModel::getContainer()->get('database')->getVar('SELECT MAX(id) FROM mini_blog LIMIT 1');
+    }
+
+    /**
+     * Get all items (at least a chunk)
+     *
+     * @param int [optional] $limit
+     * @param int [optional] $days
+     * @return array
+     */
+    public static function getTopAwesome($limit = 5, $days = 7)
+    {
+        $items = (array)BackendModel::getContainer()->get('database')->getRecords(
```

Comment:
space after cast. (maybe run php-cs-fixer fix on this module) https://github.com/FriendsOfPHP/PHP-CS-Fixer